tests/microgrid/test_client.py

```diff
@@ -407,10 +407,10 @@ async def test_meter_data(self) -> None:
             with pytest.raises(ValueError):
                 # should raise a ValueError for wrong component category
                 await microgrid.meter_data(38)
-            peekable = (await microgrid.meter_data(83)).into_peekable()
+            receiver = await microgrid.meter_data(83)
             await asyncio.sleep(0.2)
 
-        latest = peekable.peek()
+        latest = await anext(receiver)
```

Comment:
Any reason not to use `receiver.receive()`?

Reply:
no, but they are interchangeable.